### ISSUE TITLE
request.content_length

### ISSUE THREAD

**Issue by douglatornell** [npilon/pyramid_crow#1]
Thanks for creating `pyramid-crow`!

I tried it today in a Python-3.5, pyramid-1.5.7 project and got the traceback below and (obviously?) nothing sent to sentry. I commented out the over-long request body handling code:

``` python
#            if request.content_length < 2**16
#            else 'Over-long request body of {} bytes omitted'.format(
#                request.content_length
#            )
```

starting at https://github.com/npilon/pyramid_crow/blob/master/pyramid_crow/__init__.py#L26 and the exception went through to sentry.

I will try to produce a minimal case to reproduce the issue sometime this week, but wanted to raise the issue immediately in case there is something very obvious that I am overlooking.

**Traceback**

``` py3tb
Traceback (most recent call last):
  File "/home/sealinkd/miniconda/envs/sealinkd/lib/python3.5/site-packages/pyramid_debugtoolbar-2.4.1-py3.5.egg/pyramid_debugtoolbar/toolbar.py", line 192, in toolbar_tween
    response = _handler(request)
  File "/home/sealinkd/miniconda/envs/sealinkd/lib/python3.5/site-packages/pyramid_debugtoolbar-2.4.1-py3.5.egg/pyramid_debugtoolbar/panels/performance.py", line 57, in resource_timer_handler
    result = handler(request)
  File "/home/sealinkd/miniconda/envs/sealinkd/lib/python3.5/site-packages/pyramid-1.5.7-py3.5.egg/pyramid/tweens.py", line 21, in excview_tween
    response = handler(request)
  File "/home/sealinkd/miniconda/envs/sealinkd/lib/python3.5/site-packages/pyramid_crow/__init__.py", line 13, in crow_tween
    request.raven.captureException()
  File "/home/sealinkd/miniconda/envs/sealinkd/lib/python3.5/site-packages/pyramid-1.5.7-py3.5.egg/pyramid/decorator.py", line 37, in __get__
    val = self.wrapped(inst)
  File "/home/sealinkd/miniconda/envs/sealinkd/lib/python3.5/site-packages/pyramid-1.5.7-py3.5.egg/pyramid/util.py", line 52, in <lambda>
    fn = lambda this: callable(this)
  File "/home/sealinkd/miniconda/envs/sealinkd/lib/python3.5/site-packages/pyramid_crow/__init__.py", line 55, in raven_client
    client.http_context(_request_to_http_context(request))
  File "/home/sealinkd/miniconda/envs/sealinkd/lib/python3.5/site-packages/pyramid_crow/__init__.py", line 26, in _request_to_http_context
    if request.content_length < 2**16
TypeError: unorderable types: NoneType() < int()
```

**Comment by npilon** [comment]
Thank you for the bug report - I hadn't thought of `request.content_length` being `None`, but rereading the `request` documentation, it's a valid condition, even when there's body content.

**Comment by npilon** [comment]
I think this may be an "older version of pyramid" thing; I've been testing with 1.7 and can't find a way to write a test case where content length is still NULL by the time it reaches the application. https://github.com/npilon/pyramid_crow/commit/b3ffce18bde1f4e4c864f3ea5276614a2548c254 should address your problem.

**Comment by gcarothers** [comment]
Released in `0.2.1`

**Comment by douglatornell** [comment]
Thanks @npilon and @gcarothers!
I wondered if this might have been an "older version of pyramid" thing too. I can confirm though that b3ffce1 and release `0.2.1` resolve the issue for pyramid 1.5.7.
Upgrading to 1.7 is on my today :-)
